src/traces/splom/defaults.js

```diff
@@ -0,0 +1,137 @@
+/**
+* Copyright 2012-2018, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Lib = require('../../lib');
+
+var attributes = require('./attributes');
+var subTypes = require('../scatter/subtypes');
+var handleMarkerDefaults = require('../scatter/marker_defaults');
+var handleLineDefaults = require('../scatter/line_defaults');
+var PTS_LINESONLY = require('../scatter/constants').PTS_LINESONLY;
+var OPEN_RE = /-open/;
```

Comment:
Again, going to [break for symbol numbers](https://github.com/plotly/plotly.js/pull/2492#discussion_r177247877)

Reply:
-> move to https://github.com/plotly/plotly.js/issues/2507 thanks!